### DISCUSSION TITLE
View loaded configuration variables

### DISCUSSION THREAD

**Discussion by kjcsb1** [PostgREST/postgrest#3782] [Q&A] [answered]
PostgREST 12.2

I'm getting a `JWSError JWSInvalidSignature` response from PostgREST in a test environment I'm setting up. I have this working in dev but for various reasons can't use the exact same setup.

`PGRST_JWT_SECRET` is set to a 32 char alphanumeric value (not base-64 encoded).

I've validated the signature using jwt.io.

Is there a way to export the loaded configuration variables or to view logs to troubleshoot further why PostgREST can't validate the signature?

**Comment by wolfgangwalther** [comment]
There is [`postgrest --dump-config`](https://docs.postgrest.org/en/v12/references/cli.html#dump-config), which could help you.

**Reply by steve-chavez** [reply, marked as answer]
You can also use https://docs.postgrest.org/en/v12/references/admin_server.html#runtime-configuration for a running postgREST.

**Reply by kjcsb1** [reply]
Thanks that was very helpful in diagnosing the issue
